doc/encode_effort.md

```diff
@@ -25,10 +25,9 @@ The following table describes what the various effort settings do:
 | e5 | e4 + patches, local palette / local channel palette, different local RCTs | e4 + gabor-like transform, chroma from luma |
 | e6 | e5 + more RCTs and MA tree properties | e5 + error diffusion, full variable blocks heuristics |
 | e7 | e6 + more RCTs and MA tree properties | e6 + patches (including dots) |
-| e8 | e7 + more RCTs, MA tree properties and Weighted predictor parameters | e7 + Butteraugli iterations for adaptive quantization |
-| e9 | e8 + more RCTs, MA tree properties and Weighted predictor parameters, try all predictors | e8 + more Butteraugli iterations |
-| e10* | e9 + global MA tree + other whole image optimizaions | e9 + I don't know |
-| e11* | e10 + previous-channel MA tree properties, different group dimensions, exhaustively try various e10 options | N/A |
+| e8 | e7 + more RCTs, MA tree properties and Weighted predictor parameters | e7 + Butteraugli iterations for adaptive quantization and disables chunked encoding |
+| e9 | e8 + more RCTs, MA tree properties, and Gradient/Weighted predictors | e8 + more Butteraugli iterations |
```

Comment:
"Gradient/Weighted predictors" is already inherited from the e4 documentation higher in the table

Reply:
Should it be deleted, or can something can be added in its place?

Reply:
you could revert to "Weighted predictor parameters"